src/inausoft.netCLI.Tests/LogicalCommandDeserializerTests.cs

```diff
@@ -39,11 +38,56 @@ public void RegexArgumentHandler_DeserializeProperCommand_ForMultipleOptions()
             Assert.AreEqual(intOptionValue, command.IntOption);
         }
 
+        [TestMethod]
+        public void LogicalCommandDeserializer_DeserializeProperCommand_ForMultipleOptions_UsingShortNames()
+        {
+            //Arrange
+            var stringOptionValue = "sampleString";
+            var intOptionValue = 102;
+
+            var args = new string[]
+                            {
+                                "-b",
+                                "-s", stringOptionValue,
+                                "-i", intOptionValue.ToString()
```

Comment:
Some would say thah here is missing trailing coma

Reply:
I've adjusted some of them - not promising that I got all of them - but let's keep trailing comma - agreed.